testing/src/test/java/at/test/drm/ApplicationIntegrationTest.java

```diff
@@ -107,12 +109,27 @@ void shouldDeleteRelation() {
         relationService.createRelation(first, third);
         relationService.createRelation(second, third);
         final RelationLink bySourceAndIdentity =
-                relationService.findRelationBySourceObjectAndRelationIdentity(first, second);
+            relationService.findRelationBySourceObjectAndRelationIdentity(first, second);
 
         relationService.deleteRelation(bySourceAndIdentity);
 
         final RelationLink afterDelete =
-                relationService.findRelationBySourceObjectAndRelationIdentity(first, second);
+            relationService.findRelationBySourceObjectAndRelationIdentity(first, second);
         Assertions.assertThat(afterDelete).isNull();
     }
+
+    @Test
+    void shouldPrintMappedRelations() {
+        var first = new AnnotationTest();
+        var second = new AnnotationTest2();
+        var third = new AnnotationTest3();
+        dao.save(first);
+        dao2.save(second);
+        dao3.save(third);
+
+        relationService.createRelation(first, second);
+        relationService.createRelation(first, third);
+        relationService.createRelation(second, third);
+        Assertions.assertThatNoException().isThrownBy(() -> dynamicRelationsPrintService.printMappedRelations(first));
```

Comment:
Would be nice to add an assert which checks if the printed tree contains all of the testEntities